versions.yaml

```diff
@@ -77,6 +89,22 @@ versions:
       redis:
         sha256: 2139009799d21d8ff94fc40b7f36ac46699b9e1254086299f8d3b223ca54a375
         version: '5.0.5'
+    excludeTests:
+    - tests/functional_tests/redis_exporter_test.yaml
 
+  # Redis Exporter for Prometheus
+  - dir: 'exporter'
+    templateSubDir: exporter
+    repo: redis5
+    tags:
```

Comment:
Please fix tags (as we already discussed with nginx-exporter).

Reply:
Done.